python_modules/libraries/dagster-airbyte/dagster_airbyte_tests/experimental/test_asset_specs.py

```diff
@@ -20,3 +22,27 @@ def test_fetch_airbyte_workspace_data(
     actual_workspace_data = resource.fetch_airbyte_workspace_data()
     assert len(actual_workspace_data.connections_by_id) == 1
     assert len(actual_workspace_data.destinations_by_id) == 1
+
+
+def test_translator_spec(
+    fetch_workspace_data_api_mocks: responses.RequestsMock,
+) -> None:
+    with environ(
+        {"AIRBYTE_CLIENT_ID": TEST_CLIENT_ID, "AIRBYTE_CLIENT_SECRET": TEST_CLIENT_SECRET}
+    ):
+        resource = AirbyteCloudWorkspace(
+            workspace_id=TEST_WORKSPACE_ID,
+            client_id=EnvVar("AIRBYTE_CLIENT_ID"),
+            client_secret=EnvVar("AIRBYTE_CLIENT_SECRET"),
+        )
+
+        all_assets = load_airbyte_cloud_asset_specs(resource)
+        all_assets_keys = [asset.key for asset in all_assets]
+
+        # 4 tables for the connector
+        assert len(all_assets) == 1
+        assert len(all_assets_keys) == 1
```

Comment:
this comment seems completely unrelated?

Reply:
Indeed - will fix